app/javascript/topNavigation/utilities.js

```diff
@@ -0,0 +1,27 @@
+// The mapping here will never change for the duration of the current page life cycle.
+const pageLinkMapper = Object.freeze({
```

Comment:
This is the new file @ludwiczakpawel that is being imported in `/app/javascript/packs/base.js`. This also has the benefit of being testable now in the context of jest.